okio/jvm/src/main/java/okio/Buffer.kt

```diff
@@ -1034,7 +1034,7 @@ class Buffer : BufferedSource, BufferedSink, Cloneable, ByteChannel {
     val data = tail.data
     var limit = tail.limit
     data[limit++] = (s.ushr(8) and 0xff).toByte()
-    data[limit++] = (s and 0xff).toByte()
+    data[limit++] = (s                  and 0xff).toByte()
```

Comment:
This looks weird

Reply:
Umm it was an honest mistake?
<img width="466" alt="screen shot 2018-04-30 at 8 50 41 am" src="https://user-images.githubusercontent.com/133019/39428120-41c10f18-4c54-11e8-81fd-d1cf32384137.png">

Reply:
Fixed. `ushr` wanted to be infix!

Reply:
Ah, I see. I don't like parameter name hints precisely because they screw up formatting.